pkg/ccl/backupccl/restore_job.go

```diff
@@ -1665,21 +1665,22 @@ func (r *restoreResumer) notifyStatsRefresherOfNewTables() {
 
 // tempSystemDatabaseID returns the ID of the descriptor for the temporary
```

Comment:
nit: can you add some more words tying this to the logic in planning. It will help future readers to understand why its okay to just use the first system tables parentID rewrite.